test/post-deploy-utils.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2021 Adobe. All rights reserved.
```

Reply:
I think you created that file originally in 2020, it has only been copied and tweaked since then.